server/services/store/sqlstore/board.go

```diff
@@ -80,8 +80,9 @@ var boardMemberFields = []string{
 	"BM.scheme_viewer",
 }
 
-func (s *SQLStore) boardsFromRows(rows *sql.Rows) ([]*model.Board, error) {
+func (s *SQLStore) boardsFromRows(rows *sql.Rows, removeDuplicates bool) ([]*model.Board, error) {
```

Comment:
We don't seem to be using the `removeDuplicates` as `true`, why do we need to have it as a parameter?

Reply:
Yup that makes sense, will remove it. Nice catch 👍